cardea/modeling/modeler.py

```diff
@@ -1,30 +1,44 @@
-from os import listdir
-from os.path import isfile, join
+
+import os
 
 import hyperopt
 import mlprimitives
 import numpy as np
 from hyperopt import STATUS_OK, Trials, fmin, hp, tpe
 from mlblocks import MLPipeline
-from sklearn.model_selection import KFold, cross_val_score, train_test_split
+from sklearn import metrics
+from sklearn.model_selection import KFold
 
 
 class Modeler():
     '''A class responsible for executing various Machine Learning Pipelines using MLBlocks.'''
 
-    result = {}
-    problem_type = ""
-    block_instance = ""
+    problem_type = None
     primitive = []
     pipeline_dict = {}
+    scoring = None
+    data_frame = None
+    target = None
+
+    regression_scoring_function = {'explained_variance_score': metrics.explained_variance_score,
+                                   'mean_absolute_error': metrics.mean_absolute_error,
+                                   'mean_squared_error': metrics.mean_squared_error,
+                                   'mean_squared_log_error': metrics.mean_squared_log_error,
+                                   'median_absolute_error': metrics.median_absolute_error,
+                                   'r2_score': metrics.r2_score}
+    classification_scoring_function = {'accuracy': metrics.accuracy_score,
+                                       'f1': metrics.f1_score,
+                                       'precision': metrics.precision_score,
+                                       'recall': metrics.recall_score,
+                                       }
 
     def get_directory(self):
         '''Returns the path of the directory.
```

Comment:
1. Similarly, use `"""` instead of `'''`.

2. should we update the docstring content? Now it seems that it does not return "The absolute path of the directory." :thinking:, it seems a list given that you use a for-loop on line 58 `for path in mypath`

Reply:
Solves in 5ba2aa813a753fdfc895b4ba1cb2e4a82fec456e